package Day_37_Exception_Part2;

public class _02_TryCatch {
    public static void main(String[] args) {
        try{

        }catch (Exception e){

        }

    }
}
